azure/manifest.yaml

```diff
@@ -0,0 +1,19 @@
+applicationName: ondat
+publisher: "Ondat"
+description: "Cloud Native persistent storage for containers - a test package for the marketplace CNAB structure"
+version: 2.8.1 #Must be semver # version of application (version tag for the cnab bundle)
```

Comment:
Could we set this to 2.9.0?

Reply:
I['ve actually removed this file entirely, i don't see a reason to keep it on main & the azure-charts branch